recipes/java-jdk/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "8.25.17" %}
+
+package:
+  name: java-jdk
+  version: 8.25.17
+
+# Cannot use source section because of cookies blocking download
+#source:
+#  fn: jre-8u25-linux-x64.tar.gz
+#  url: http://download.oracle.com/otn-pub/java/jdk/8u25-b17/jre-8u25-linux-x64.tar.gz
+
+build:
+  number: 0
```

Comment:
That's unfortunate.

The next step kind of sucks and may need to be done locally to iterate quickly. Try a mixture of adding binary files to [`binary_has_prefix_files`]( http://conda.pydata.org/docs/building/meta-yaml.html#binary-has-prefix-files ) and text files to [`has_prefix_files`]( http://conda.pydata.org/docs/building/meta-yaml.html#text-files-with-prefix-files ).